### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kgevorkyan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,36 @@
-Current repo contain the set of tests, adopted for execution in [saveourtool](https://saveourtool.com/)
+Current repo contain the set of tests, adopted for execution in [saveourtool](https://saveourtool.com/) platform.
```
Comment (orchestr7) [inline README.md:1]: Save Cli? link

File: README.md [hunk 2]
```diff
@@ -1,6 +1,36 @@
-Current repo contain the set of tests, adopted for execution in [saveourtool](https://saveourtool.com/)
+Current repo contain the set of tests, adopted for execution in [saveourtool](https://saveourtool.com/) platform.
 
-For local execution you need to run following commands **in the root of project**
+### Local run
 
-    ./setup.sh
-    ./save-0.3.9-linuxX64.kexe . --log all --report-type json --result-output file
\ No newline at end of file
+1) Extract the files from `executable` directory to the root of project.
```
Comment (nulls) [inline README.md:15]: maybe refactor it and move all tests to subfolder?

Comment (orchestr7) [reply README.md:15]: +++++++

Comment (orchestr7) [reply README.md:15]: We are right now arguing about this

Comment (nulls) [reply README.md:15]: good luck then!)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,43 @@
...
+### Local run
 
-    ./setup.sh
-    ./save-0.3.9-linuxX64.kexe . --log all --report-type json --result-output file
\ No newline at end of file
+Just execute
+
+    ./run_local.sh
+
+It will do the following steps:
```
Comment (orchestr7) [inline README.md:13]: put spoiler here

File: README.md [hunk 4]
```diff
@@ -1,6 +1,43 @@
...
+
+The `libAnalyzer.so`, provided in the `executable` directory of repository will create
+`.sarif` report files for each test, after what all reports will be collected in `reports` directory.
+
+The `save` will produce the report in `save-reports` directory in `json` format.
+
+It will contain detailed information about matched warnings, where expected warnings were taken from `.sarif` reports,
+created by `libAnalyzer.so` and actual warnings extracted directly from the test files, according pattern `expectedWarningsPattern`.
+
+**Note:** all warnings should be provided in tests like a comments:
```
Comment (orchestr7) [inline README.md]: Move note to anchor

File: run_local.sh [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+sudo cp -rp executable/* .
```
Comment (nulls) [inline run_local.sh:3]: do we need a sudo to copy files?

Comment (kgevorkyan) [reply run_local.sh:3]: Dont quite sure, but left it for `-p` option, which resposible for saving the righs of files (such as chmod and so on)

**Pull request merged**